Allow the Dokumentversjon#filnavn field to be set

ivaylomitrev opened the pull request.

ivaylomitrev committed:
Allow the Dokumentversjon#filnavn field to be set

dkumanov commented on src/main/java/com/documaster/rms/noark/ws/noarkentities/Dokumentversjon.java:
```diff
@@ -28,8 +28,8 @@ public class Dokumentversjon extends NoarkEntityBase<Dokumentversjon> {
 
-	@JsonIgnore
 	private String filnavn;
+	private boolean serializeFilnavn;
```
Since we can't really set the field to null, I think a simple setter will do the job.

ivaylomitrev replied:
Fixed w/ https://github.com/documaster/ws-client-n5-java/commit/10bb4a879b8379bd1f6a5a1df6c86499d9235861. I force-pushed, because the end result is much smaller than what 2 commit would have produced 😅

ivaylomitrev committed:
Set @JsonIgnore on Dokumentversjon getters

This is needed because Jackson is configured to serialize fields based
on the getters and the @JsonIgnore annotations we had on the fields
themselves were actually ignored since the dawn of time. This led to
these fields being sent to the backend when a document version was
updated which was, in turn, rejected by said backend which does not
allow these fields to be updated.

ivaylomitrev commented:
As discussed internally, I have disallowed the serialization of the dokumentversjon fields by annotating the _getters_ with @JsonIgnore since the annotation placed on the fields had no effect due to Jackson configuration: https://github.com/documaster/ws-client-n5-java/pull/46/commits/d01c578053b04dbda50176004d0c79e1eb17a88c

dkumanov commented:
No comments after the latest changes. My approval stands :green_circle:
